src/scandeval/dataset_configs.py

```diff
@@ -562,6 +562,19 @@ def get_dataset_config(dataset_name: str) -> DatasetConfig:
     max_generated_tokens=3,
 )
 
+DUTCH_COLA_CONFIG = DatasetConfig(
+    name="dutch-cola",
+    pretty_name="a linguistic acceptability dataset for Dutch, Dutch CoLA, inspired by the original CoLA dataset",
+    huggingface_id="BramVanroy/dutch-cola",
+    task=LA,
+    languages=[NL],
+    prompt_prefix="Hieronder staan zinnen en of ze grammaticaal correct ('ja') of incorrect ('nee') zijn.",
+    prompt_template="Zin: {text}\nGrammaticaal correct: {label}",
+    prompt_label_mapping=dict(correct="ja", incorrect="nee"),
+    num_few_shot_examples=12,
+    max_generated_tokens=3,
+)
```

Comment:
I would make the dataset unofficial to begin with, until we have tested whether the performance differentiation is as good / better than ScaLA-nl.

```suggestion
    max_generated_tokens=3,
    unofficial=True,
)
```